src/platform/intel/cavs/platform.c

```diff
@@ -336,13 +336,15 @@ static void platform_init_hw(void)
 	io_reg_write(DSP_INIT_IOPO,
 		IOPO_DMIC_FLAG | IOPO_I2S_FLAG);
 
+#ifndef __ZEPHYR__
 	io_reg_write(DSP_INIT_ALHO,
 		ALHO_ASO_FLAG | ALHO_CSO_FLAG);
 
 	io_reg_write(DSP_INIT_LPGPDMA(0),
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
 	io_reg_write(DSP_INIT_LPGPDMA(1),
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
+#endif
 }
 
```

Comment:
@kv2019i did you guys moved the DMA drivers to zephyr? Care to let me know the path in zephyr where this happens?

Reply:
@dbaluta Ack, all DMA drivers (HD-DMA and DW-DMA) moved now. They are in work/zephyr/drivers/dma/